data_steward/cdr_cleaner/cleaning_rules/remove_non_matching_participant.py

```diff
@@ -67,6 +67,17 @@ def exist_participant_match(ehr_dataset_id, hpo_id):
         bq_utils.get_table_id(hpo_id, PARTICIPANT_MATCH), ehr_dataset_id)
 
 
+def exist_identity_match(table_id, dataset_id):
+    """
+    This function checks if the hpo has valid the identity_match table
+
+    :param table_id:
+    :param dataset_id:
+    :return:
+    """
+    return bq_utils.table_exists(table_id, dataset_id)
```

Comment:
`bq_utils` will be deprecated. You could use [this example](https://cloud.google.com/bigquery/docs/samples/bigquery-table-exists#bigquery_table_exists-python) for the same check. This will require creating a client but it's not a big change.